android/app/src/main/java/com/keyman/keyman_browser/MainActivity.java

```diff
@@ -1,6 +1,93 @@
 package com.keyman.keyman_browser;
 
+import java.util.List;
+import android.os.Bundle;
+import io.flutter.plugin.common.MethodCall;
 import io.flutter.embedding.android.FlutterActivity;
+import io.flutter.embedding.engine.FlutterEngine;
+import io.flutter.plugin.common.MethodChannel;
+
+import android.content.BroadcastReceiver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import androidx.annotation.NonNull;
+import android.util.Log;
+
+import android.os.Handler;
+import android.provider.Settings;
+import android.database.ContentObserver;
+
 
 public class MainActivity extends FlutterActivity {
+  private static final String CHANNEL = "com.example.font_channel";
```

Comment:
Maybe this can be moved inside configureFlutterEngine if that's the only use.
Could the string can be arbitrary, could it be "com.keyman.browser.font_channel"?

Reply:
ah, it probably needs to match the string in address_bar.dart